dts/bindings/clock/intel,cavs-shim-clkctl.yaml

```diff
@@ -3,7 +3,7 @@
 
 description: Intel cAVS clock controlling related constants.
 
-compatible: "intel,cavs-shim-clkctl"
+compatible: "intel,adsp-shim-clkctl"
```

Comment:
You will also need to rename the yaml file.

Reply:
Renamed!